app/src/main/java/org/stepic/droid/di/home/HomeModule.kt

```diff
@@ -0,0 +1,21 @@
+package org.stepic.droid.di.home
+
+import dagger.Module
+import dagger.Provides
+import io.reactivex.Scheduler
+import org.stepic.droid.core.presenters.HomeStreakPresenter
+import org.stepic.droid.di.qualifiers.BackgroundScheduler
+import org.stepic.droid.di.qualifiers.MainScheduler
+import org.stepic.droid.preferences.SharedPreferenceHelper
+import org.stepic.droid.web.Api
+
+@Module
+class HomeModule {
+    @Provides
+    @HomeScope
+    internal fun provideStreakPresenter(@BackgroundScheduler backgroundScheduler: Scheduler,
```

Comment:
it is not needed, just use `@HomeScope` on `HomeStreakPresenter` class and dagger will resolve it automatically